integrations/spark/openhouse-spark-runtime/build.gradle

```diff
@@ -28,6 +28,7 @@ configurations {
     exclude(group: 'org.mapstruct')
   }
   shadow.extendsFrom implementation
```

Comment:
pom file seem to again have antlr runtime, how does this exclusion help?
![image](https://github.com/user-attachments/assets/0e48ccb9-3a75-4005-a7c4-c72c0bd3ad16)


Also, can we give compileOnly a shot for antlr?

Reply:
Sure, let me try with compileOnly for antlr.
`antlr4-runtime` is added as runtime dependency only as per [this](https://github.com/linkedin/openhouse/blob/main/integrations/spark/openhouse-spark-runtime/build.gradle#L36) code and it seems no issue with this dependency. The issue reported was for antlr4.

Reply:
@HotSushi build actually fails after changing `antlr` -> `compileOnly`.